public/examples/download/downloadApp.js

```diff
@@ -328,16 +324,21 @@
                 myChart.setOption(option, true);
         }
 
+        urls.length = 0;
 
+        var baseUrl = (version === 'IPv6') ? testPlan.baseUrlIPv6NoPort : testPlan.baseUrlIPv4NoPort;
 
-        var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
+        for (var i = 0; i < ports.length; i++) {
```

Comment:
so this will be 6 * 6 = 36 urls?
Is that what you intended or you wanted to have  6 of the same url with different ports?